x-pack/plugins/infra/public/alerting/common/components/threshold.tsx

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { Chart, Metric, Settings } from '@elastic/charts';
+import { EuiIcon, EuiPanel, useEuiBackgroundColor } from '@elastic/eui';
+import type { PartialTheme, Theme } from '@elastic/charts';
+import { i18n } from '@kbn/i18n';
+import { Comparator } from '../../../../common/alerting/metrics';
+
+export interface ChartProps {
+  theme: PartialTheme;
+  baseTheme: Theme;
+}
+
+export interface Props {
+  chartProps: ChartProps;
+  comparator: Comparator;
+  threshold: number;
+  title: string;
+  value: number;
+  valueFormatter: (d: number) => string;
+}
+
+export const Threshold = ({
+  chartProps: { theme, baseTheme },
+  comparator,
+  threshold,
+  title,
+  value,
+  valueFormatter,
+}: Props) => {
+  const color = useEuiBackgroundColor('danger');
+
+  return (
+    <EuiPanel
+      paddingSize="none"
+      style={{
+        height: '100%',
+        overflow: 'hidden',
+        position: 'relative',
+        minWidth: '100%',
+      }}
+      hasShadow={false}
+      data-test-subj={`threshold-${threshold}-${value}`}
+    >
+      <Chart>
+        <Settings theme={theme} baseTheme={baseTheme} />
+        <Metric
+          id="1"
```

Comment:
**nit:** why is this id static to 1? Shouldn't be something configurable from the parent component?